### ISSUE TITLE
[Bug Report] [Critical] NBT Data for Modded Blocks Gets Erased/Reset

### ISSUE THREAD

**Issue by bauerbach** [Amulet-Team/Amulet-Map-Editor#626]
### Current Behaviour:
All NBT data from modded blocks is erased when a world is saved in Amulet. I used Amulet on a modded world (with a bunch of Futurepack Sorters), and now all configurations for the Sorters was erased, forcing me to reconfigure them all over again.

### Expected behavior:
NBT data for modded blocks should be saved when saving a world with Amulat,

### Environment:

- OS: Windows
- Minecraft Platform: Java 
- Minecraft Version: 1.16.5
- Amulet Version: 0.8.15

### Additional context
Vanilla blocks do not lose their inventories or other NBT data when the world is saved after changes are made in Amulet.

**Comment by Michael5156** [comment]
Have you noticed modded blocks getting removed entirely?

**Comment by bauerbach** [comment]
> Have you noticed modded blocks getting removed entirely?

Yep! I forgot to mention that blocks such as the AND gates and NOT gates from Project:Red are completely removed when saving.

**Comment by Griefed** [comment]
This sounds like an accidental feature. 😁
If removing modded blocks/items could not be removed entirely but instead be moved to its own feature, that would be much appreciated.
